cmd/flags.go

```diff
@@ -27,7 +27,7 @@ const (
 	defaultMaxConcurrentManifestReconciles = 1
 	defaultMaxConcurrentKymaReconciles     = 1
 	defaultMaxConcurrentWatcherReconciles  = 1
-	defaultIstioGatewayName                = "lifecycle-manager-watcher-gateway"
```

Comment:
I clarified with. SRE. Normally, ARGO takes care of removed those, if not we we need to contact SREs and tell them what to remove. This would be a one time action